CHANGELOG.md

```diff
@@ -4,19 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [2.2.0] - 2018-06-07
 
 ### Added
 
 - Added `prediction_gradient` method for understanding the impact of different
   features in MLPs with dense inputs.
 - Added support for SELU activations with alpha dropout.
+- Added sample weights for the `FMClassifier`.
```

Comment:
We should also include

```
- Added `FMRegressor`.
```

right?

https://github.com/civisanalytics/muffnn/pull/93